docs/visual-basic/misc/all-field-widths-except-the-last-element-must-be-greater-than-zero.md

```diff
@@ -22,7 +22,7 @@ All field widths, except the last element, must be greater than zero. A field wi
 -   Set the field width to the correct length.  
   
 ## See Also  
- [TextFieldParser.SetFieldWidths Method](http://msdn.microsoft.com/en-us/958fed9f-e0f3-4fc5-83b4-386156bdf036)  
- [TextFieldParser.FieldWidths Property](http://msdn.microsoft.com/en-us/c6985360-60c6-494e-89e7-43b6b73f2597)  
+ <xref:Microsoft.VisualBasic.FileIO.TextFieldParser.SetFieldWidths>  
```

Comment:
The link is broken; it should be \<xref:Microsoft.VisualBasic.FileIO.TextFieldParser.SetFieldWidths%2A?displayProperty=nameWithType>